app/monitoring/Metrics.scala

```diff
@@ -1,11 +1,9 @@
 package monitoring
 
-import com.amazonaws.regions.{Region, Regions}
 import com.gu.monitoring.CloudWatch
 import configuration.Config
 
 trait Metrics extends CloudWatch {
-  val region = Region.getRegion(Regions.EU_WEST_1)
```

Comment:
Nope it was removed in: https://github.com/guardian/membership-common/pull/413/files